src/package-lock.json

```diff
@@ -26,6 +26,9 @@
         "js-beautify": "^1.14.6",
         "tv4": "^1.3.0",
         "yaml": "^2.1.1"
+      },
```

Comment:
Was this change intentional?

Reply:
The change was not made by me. It was made three months ago.

https://github.com/SchemaStore/schemastore/commit/8ef2246ceb93aae34cd5f17fb09f628250e0d45c#diff-cc5a2f170768969f124108ad3be7fdbcbed774c11774d99a87a3a78fef4ab97bL29-L47

https://github.com/SchemaStore/schemastore/blob/8ef2246ceb93aae34cd5f17fb09f628250e0d45c/src/package.json#L40-L48

Reply:
It was not intentional. It was automatically generated after I ran `npm install`.

Reply:
I've  finished it.